client/command/filesystem/download.go

```diff
@@ -47,14 +54,22 @@ func DownloadCmd(ctx *grumble.Context, con *console.SliverConsoleClient) {
 
 	src := remotePath
 	fileName := filepath.Base(src)
-	dst, _ := filepath.Abs(localPath)
-	fi, err := os.Stat(dst)
-	if err != nil && !os.IsNotExist(err) {
-		con.PrintErrorf("%s\n", err)
-		return
-	}
-	if err == nil && fi.IsDir() {
-		dst = path.Join(dst, fileName)
+	var dst string
+
+	if ctx.Flags.Bool("loot") {
+		// Put something in here for when the message below is output
+		dst = "loot"
+	} else {
+		// If this download is not being looted, make sure the local path exists
+		dst, _ := filepath.Abs(localPath)
```

Comment:
Doesn't cost anything to check the `error` returned by `filepath.Abs` here, so we should probably do it just in case :)

Reply:
Sounds good :)